### PR TITLE
cloud talent: Updated V4 Beta 1 docs

### PR DESCRIPTION
**Pull request by hilts-vaughan**
Regenerated the docs via synthtool

These are currently not right on the production site. Is there any plans to get these into master?

### CODE REVIEW

**Comment by bshaffer** [comment]
See #971, essentially these samples were pushed to the `talent-v4better1-samples` branch and have not been reviewed. The branch was published to our documentation and that's all I know. .

The issue is some of the sample generation needs to be cleaned up. The `$opts` array for example, is not the way we want to handle this. So until we can find someone to work with to make the required changes, I do not think we can merge it into `master`. However, I am more than happy to work with someone to make the required changes, I do not think it will be a large effort.

**Comment by bshaffer** [comment]
I'd like to see these samples [more closely match our other samples](https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/bigquery/api/src/browse_table.php#L25) before merging into master.

**Comment by hilts-vaughan** [comment]
Could you summarize the changes you want to see? I imagine a lot of it comes down to the generation tool being used.

**Comment by bshaffer** [comment]
@hilts-vaughan I've been told the generation tool is not going to be maintained going forward, so I think it would make more sense to just manually change the files here.

**Comment by bshaffer** [comment]
closing these because PHP samples have been removed for this API
